proposals/compatibility-mode.md

```diff
@@ -68,33 +70,47 @@ Each `GPUColorTargetState` in a `GPUFragmentState` must have the same `blend.alp
 
 **Justification**: OpenGL ES 3.1 does not support indexed draw buffer state.
 
+Feature: `indexed-draw-buffers`. Supported via `GL_EXT_draw_buffers_indexed`, `GL_OES_draw_buffers_indexed`, or OpenGL ES 3.2.
```

Comment:
This uses GL terminology that we don't use. Maybe `heterogeneous-color-target-states` or something, I don't have a great suggestion

Reply:
`per-color-target-fragment-state` (my favourite i think)
`per-color-target-blend-state`
`per-color-target-state`
`fragment-state-per-color-target`
`per-color-attachment-fragment-state`
`per-color-attachment-blend-state`